save-cli/src/nativeMain/kotlin/org/cqfn/save/cli/SaveCliRunner.kt

```diff
@@ -7,6 +7,8 @@ package org.cqfn.save.cli
 import org.cqfn.save.core.Save
 
 fun main(args: Array<String>) {
+    val cliArguments = createConfigFromArgs(args)
```

Comment:
I guess these args should be used when creating `Save`?

Reply:
oh, fixed